apps/aecore/test/aec_miner_tests.erl

```diff
@@ -32,30 +32,29 @@ miner_test_() ->
                                  meck:passthrough([]) div 2560
                          end),
              aec_test_utils:mock_time(),
+             ok = application:ensure_started(gproc),
+             TmpKeysDir = aec_test_utils:aec_keys_setup(),
              {ok, _} = aec_tx_pool:start_link(),
              {ok, _} = aec_chain:start_link(aec_block_genesis:genesis_block()),
-             TmpKeysDir = mktempd(),
              ok = application:ensure_started(crypto),
-             {ok, _} = aec_keys:start_link(["mypassword", TmpKeysDir]),
              {ok, _} = ?TEST_MODULE:start_link([{autostart, true}]),
              TmpKeysDir
      end,
      fun(TmpKeysDir) ->
              ok = ?TEST_MODULE:stop(),
-             ok = aec_keys:stop(),
              ok = aec_chain:stop(),
              ok = aec_tx_pool:stop(),
-             ok = application:stop(crypto),
-             {ok, KeyFiles} = file:list_dir(TmpKeysDir),
-             %% Expect two filenames - private and public keys.
-             [_KF1, _KF2] = KeyFiles,
-             lists:foreach(
-               fun(F) ->
-                       AbsF = filename:absname_join(TmpKeysDir, F),
-                       {ok, _} = {file:delete(AbsF), {F, AbsF}}
-               end,
-               KeyFiles),
-             ok = file:del_dir(TmpKeysDir),
+             ok = application:stop(gproc),
+             aec_test_utils:aec_keys_cleanup(TmpKeysDir),
+             %% {ok, KeyFiles} = file:list_dir(TmpKeysDir),
```

Comment:
This can be deleted (no need to update PR though).

Reply:
Agreed.

Reply:
There's an attempt to delet `TmpKeysDir` on line 61 as well (which likely never worked to begin with since it uses `file:delete/1` on a directory) ...